src/utils/Configuration.ts

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
The other services have used a slightly different model for config - using a config.yml, and Configuration.ts just parses and splits up the pieces of config. Not sure if you have a strong preference, but I was inclined to go for consistency.

Reply:
The idea of config as yml, as used in3330 is definitely better than the approach used in defects.  Additionally there are also models which I found are just code repetitions across the lambdas, contrary to the DRY principles.
Probably as next stage we can have a common npm repo which will contain these common features/functionalities and can be directly sourced across the lambdas so we can facilitate the DRY principles .. 

On the same note I was thinking of the pattern used in jest for mocking dynamodb endpoint as facilitated by jest framework.

For these reasons I've not tried to optimise these common features as part of this commit, but to use existing and await for an agreed pattern which leverages the agreed ideas and principles across all lambdas